### PR TITLE
Updates for Open Beta Program

### PR DESCRIPTION
**Pull request by r2liquibase**
Rolling out the Open Beta Program.

### CODE REVIEW

File: beta/session-1.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Read through [this quick start tutorial.](https://titan-data.io/getting-started)
 
 ## Step 5: Download Liquibase Data extension and add to your system path.
 
-1. Download [Liquibase Data extension.](https://github.com/liquibase/liquibase-data/releases/tag/v0.0.1) 
+1. Download the latest [Liquibase Data extension.](https://github.com/liquibase/liquibase-data/releases) 
 2. Unzip the downloaded release.
```
Comment (mcred) [inline beta/session-1.md:62]: It's a jar now, not longer need to extract a zip. We fixed this right after 0.0.1

File: beta/session-3.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ Tags help you assign human-friendly names to specific commits. They can also be
 
 To create a tag, identify your last commit id with the following:
 
-    liquibase data log --repository=<your-repos-name>
+    liquibase data log --repository=myrepos
 
 Once you have the commit id, apply a tag with the following command:
 
-    liquibase data datatag --repository=<your repos name> --commit=<commit id> --tags=<tag name>
```
Comment (mcred) [inline beta/session-3.md]: This should just be `liquibase data tag`.

**Review by mcred — CHANGES_REQUESTED** [review]

File: beta/session-1.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+> If you have previously installed Docker, verify that you are running 3.2.0 or higher. We have tested and require usage of Docker Desktop Community 3.2.0 or higher for this Beta. This includes the latest version, 3.6.0.
+
+**SYSTEM SPECIFIC SETUP**
+
+- On **Windows**, disable "Use the WSL2 Based Engine" in your Docker settings on Windows. We currently do not support Docker running via WSL2. You can always change this setting back later.
+
+![Image of Windows WSL2 Settings](images/docker_wsl2_settings.png)
+
+- On **Mac**, disable "Use gPRC FUSE for file sharing".
```
Comment (mcred) [inline beta/session-1.md:28]: Should we also add the file sharing steps? This should prevent the issue you ran into.

File: beta/session-1.md (hunk 1 above)
Comment (r2liquibase) [reply beta/session-1.md]: fixed!

File: beta/session-1.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+## Step 12: Get the most recent commit from the repository
+
+Once you have seen the initial database version, checkout the latest commit to see your changes. You should now see a single table in your database tool.
+
+You will need the second commit id. That is listed with the previous command, `liquibase data log --repository=myrepos`
+
+Execute this command:
+
+    liquibase data checkout --commit=<commit id>--repository=myrepos
```
Comment (mcred) [inline beta/session-1.md:184]: `--commit=<commit id>--repository=myrepos` needs a space between parameters

File: beta/session-3.md (hunk 2 above)
Comment (r2liquibase) [reply beta/session-3.md]: fixed!

File: beta/session-3.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Tags help you assign human-friendly names to specific commits. They can also be "floating tags" so that you are able to mark a commit as a "golden database" for standardized testing and development. Tags in Titan are strings with no spaces. Multiple tags (e.g., -t foo -t bar) are acceptable. Tags are applied to a previously applied commit.
+
+To create a tag, identify your last commit id with the following:
+
+    liquibase data log --repository=myrepos
+
+Once you have the commit id, apply a tag with the following command:
+
+    liquibase data datatag --repository=myrepos --commit=<commit id> --tags=<tag name>
```
Comment (mcred) [inline beta/session-3.md]: `liquibase data tag`

**Review by mcred — CHANGES_REQUESTED** [review]

File: beta/session-1.md (hunk 3 above)
Comment (r2liquibase) [reply beta/session-1.md:28]: Absolutely. I added it to Session 3 where the diff stuff is. Don't want to clutter the first session any more.

File: beta/session-1.md (hunk 4 above)
Comment (r2liquibase) [reply beta/session-1.md:184]: fixed!

**Comment by r2liquibase** [comment]
> What do you think about adjusting the docs to that early on `myrepos` is added to `liquibase.properties` as `repository= myrepos`. This will prevent typing this parameter repeatedly.

Holy shit. Didn't know we could do that! Ha!!!

File: beta/session-1.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+Think of this step as your `git init` for the versioned database. The following command will start a PostgreSQL Docker container and allow Titan to manage versions for the container. 
+
+> Your repository name can be anything you would like. We are going to use `myreps`. Feel free to change that to whatever you would like!
+
+Execute this command: 
+
+    liquibase data run --name=myrepos --env=POSTGRES_PASSWORD=secret --image=postgres
+
+NOTE: The `--name` argument should match the repository name you listed in `liquibase.properties`.
```
Comment (mcred) [inline beta/session-1.md:98]: I think adding the repository argument to the properties should happen after this step. I don't feel strongly about it, though.

**Review by mcred — CHANGES_REQUESTED** [review]

**Review by mcred — APPROVED** [review]

File: beta/session-1.md (hunk 6 above)
Comment (r2liquibase) [reply beta/session-1.md:98]: ok, let's ship and see if people complain. cool?

**Pull request merged**